packages/discord.js/typings/index.d.ts

```diff
@@ -2815,16 +2815,14 @@ export class GuildChannelManager extends CachedManager<Snowflake, GuildBasedChan
   ): Promise<MappedGuildChannelTypes[T]>;
   /** @deprecated See [Self-serve Game Selling Deprecation](https://support-dev.discord.com/hc/en-us/articles/4414590563479) for more information */
   public create(
```

Comment:
You need to remove the `name` parameter from the overload above this one (I can't comment there)